tests/unit/components/sl-modal-test.js

```diff
@@ -318,3 +258,7 @@ test( 'aria-labelledby is set', function( assert ) {
         this.$().attr( 'aria-labelledby' )
     );
 });
+
+// These tests require the ember-stream/streamService to be available to test
```

Comment:
I know we previously talked about this and decided to skip this test but now that I'm thinking about things even if we knew how to instantiate and use the `ember-stream/streamService` it would still not be the correct approach in the test to use it.  Conceptually, this may just as well be a database, API or some other service.  The point being, we do not want to test any of it, but rather our interaction with its expected existence.  We should be mocking this service for our tests, just like we were mocking the Modal Service.  Now the mocking and registering it for usage may still require us to skip the test for now, but if so then we should refine the comment to indicate this rather than what it currently says.